releasenotes/notes/refactor-reconstruct-45e00c3df1bdd4ff.yaml

```
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    :func:`reconstruct_expectation_values` now takes a :class:`~qiskit.primitives.SamplerResult` instance or a dictionary mapping partition labels to :class:`~qiskit.primitives.SamplerResult` instances as input to the first positional arg, ``results``. ``results`` replaces the old ``quasi_dists`` positional arg. The :class:`~qiskit.primitives.SamplerResult` instances are expected to contain the number of QPD bits used in each circuit input to the Sampler. This should be specified in the ``num_qpd_bits`` field of the experiment result metadata.
```

should check sphinx for ``` :func:`reconstruct_expectation_values` ```, to make sure it works fine without being prefixed with a period

---

I built the Sphinx docs locally, and yes, the period is necessary for it to work.

```suggestion
    :func:`.reconstruct_expectation_values` now takes a :class:`~qiskit.primitives.SamplerResult` instance or a dictionary mapping partition labels to :class:`~qiskit.primitives.SamplerResult` instances as input to the first positional arg, ``results``. ``results`` replaces the old ``quasi_dists`` positional arg. The :class:`~qiskit.primitives.SamplerResult` instances are expected to contain the number of QPD bits used in each circuit input to the Sampler. This should be specified in the ``num_qpd_bits`` field of the experiment result metadata.
```